python_src/src/lamp_py/migrations/versions/metadata_prod/07903947aabe_initial_changes.py

```diff
@@ -40,6 +45,52 @@ def upgrade() -> None:
         unique=False,
         postgresql_where=sa.text("rail_pm_processed = false"),
     )
+
+    # pull metadata from the rail performance manager database into the
```

Comment:
i think this works how we would like. it uses a test based query to trya and select metadata from the rail performance manager and eats an exception iff its caused by an "undefined table" error, i.e. the metadata log table does not exist.